src/components/global-navigation/GlobalNavigationItemStyles.tsx

```diff
@@ -20,8 +20,9 @@
 
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
+import * as radixNavigationMenu from '@radix-ui/react-navigation-menu';
 
-export const StyledWrapper = styled.div`
+export const StyledNavMenuItem = styled(radixNavigationMenu.Item)<{ 'data-selected': boolean }>`
```

Comment:
We shouldn't have to type the `data-selected` prop, especially has it's not used as a prop but as a HTML data attribute.

Reply:
I wanted to force its use to not forget it

Reply:
ah yeah that can make sense